src/components/comment/comment.test.ts

```diff
@@ -44,17 +59,44 @@ test(
     await expect(form).not.toBeVisible();
 
     await user.click(replyButton);
+    expect(await getAccessibilityTree(body)).toMatchInlineSnapshot(`
+      article "Test author name said:"
+        banner
+          heading "Test author name said:"
+            text "Test author name"
+            text "said:"
+        text "Test content"
+        contentinfo
+          link "Permalink to Test author name’s Jan 1, 2000 comment"
+            text "Permalink to Test author name’s"
+            text "Jan 1, 2000"
+            text "comment"
+          form "Reply to Test author name"
+            heading "Reply to Test author name"
+              text "Reply to Test author name"
+            text "Please be kind, courteous and constructive.
+                      You may use simple HTML or"
+            link "Markdown"
+              text "Markdown"
+            text "in your comments.
+                      All fields are required."
+            text "Reply"
+            textbox "Reply" (focused)
+              ↳ description: "Please be kind, courteous and constructive. You may use simple HTML or Markdown in your comments. All fields are required."
+            text "Name"
+            textbox "Name"
+            text "Email"
+            textbox "Email"
+            checkbox "Notify me of follow-up comments by email."
+            text "Notify me of follow-up comments by email."
+            button "Submit Reply"
+            button "Cancel"
+    `);
 
     // Updated state: reply form is no longer hidden
     await expect(form).toBeVisible();
     // Reply button is hidden
     await expect(replyButton).not.toBeVisible();
-    // The first input or textarea should be focused.
-    // (In practice this will always be a textarea, but if the form is
-    // changed to lead with an input in the future, and that input is selected,
-    // this test should still pass.)
```

Comment:
It's less clear to me now what this test is checking for. I wonder if we should add a comment above the snapshot explaining that we're interested in checking that the first focusable element in the form should be focused?

Reply:
Yeah sure 👍 . It is also checking that all the parts of the form are rendered with the intended text + roles + structure